### DISCUSSION TITLE
BitTorrent protocol v2 support in your rtorrent fork

### DISCUSSION THREAD

**Discussion by jonesnc** [jesec/flood#489] [Q&A] [answered]
Now that qBittorrent 4.4.0 is out and [has support for BitTorrent protocol v2](https://github.com/qbittorrent/qBittorrent/blob/a3be239a6082cc17521efc3663e19359d5d64586/Changelog#L4), do you think your fork of `rtorrent` would ever support BitTorrent v2?

**Comment by jesec** [comment, marked as answer]
See jesec/rtorrent#1.
